### PR TITLE
Portfolio: finish mobile version

### PR DESCRIPTION
**Pull request by AlselwiAisha**
### **Description:**

- Add  Projects section.
- Add About section.
- Use Flexbox.
- Use  CSS Grid.
- Use ul and li for categories in "Projects" section. Html, CSS, and JavaScript should be list items.
- Use ul and li for categories in "About myself" section. Languages, Frameworks, and Skills should be list items.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -35,6 +36,131 @@ <h1 class="headline-h1 main-color">Aisha <br>Alselwi</h1>
...
+<div class="project">
+<img src="./Images/1.png" alt="">
+<div class="proj-card">
+  <h2 class="proj-name white">Project name goes here</h2>
+  <ul class="proj-langs">
+     <li><a href=""> HTML/CSS</a></li>
+     <li><a href=""> Ruby on Rails</a></li>
+     <li><a href=""> JavaScript</a></li>
+  </ul>
+  <button class="btn">See this project <i class='bx bx-arrow-back'></i></button>
```
Comment (Zuhaib042) [inline index.html:51]: - [ ] Kindly consider adding `display: flex; and align-items: center;` to `btn` class to align text and arrow icon as it is in the figma design.

File: index.html [hunk 2]
```diff
@@ -35,6 +36,131 @@ <h1 class="headline-h1 main-color">Aisha <br>Alselwi</h1>
...
+      <ul>
+          <li> Database Management</li> 
+          <li> CLI</li>
+          <li> API Design</li>
+          <li> Version Control</li>
+          <li> Web Development</li>
+        </ul>
+      </div>
+      <img src="./Images/ic_skills.png" alt="">
+     </div>
```
Comment (Zuhaib042) [inline index.html]: - [ ] Kindly structure this section as technology categories ("Languages", "Frameworks" and "Skills") are in a `ul`, each of them in a `li`, that will make your design semantically better.  the ul and li's you are taking inside languages or skills or frameworks is correct. I am suggesting that you take those three headings in a ul as well.

**Review by Zuhaib042 — CHANGES_REQUESTED** [review]
Hi @AlselwiAisha 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!


## Highlights
✅ Very good design
✅ Implemented most of the requirements

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by nedjwab — APPROVED** [review]
# 🟢 Project Approved! 🏆

## Hi  @AlselwiAisha     👋

Your project is complete! ✔️✔ , I loved your work, it looks great 🔥🔥🔥🔥 🏁

Your code was very professional 💯

There is nothing else to say other than... _it's time to merge it_ :shipit:
Congratulations! 🎉

 
## Highlights 🌟

- ✔️ Descriptive commmit messsages
- ✔️ Descriptive PR description
- ✔️ No linter Errors

- The code is top-notch! 💯 and have a huge potential.

Cheers and Happy coding!👏👏👏

## Important Notes 📝

- Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
- Please, remember to tag me in your question so I can receive the notification.


_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**